Vostok.Logging.Microsoft.Tests/VostokLoggerProvider_Tests.cs

```diff
@@ -8,9 +8,10 @@
 using NUnit.Framework;
 using Vostok.Logging.Abstractions;
 using Vostok.Logging.Abstractions.Values;
-using Vostok.Logging.Abstractions.Wrappers;
+using Vostok.Logging.Microsoft.Tests.Helpers;
 using LogLevel = Vostok.Logging.Abstractions.LogLevel;
 using MsLogLevel = Microsoft.Extensions.Logging.LogLevel;
+using VostokLogEvent = Vostok.Logging.Abstractions.LogEvent;
 
 namespace Vostok.Logging.Microsoft.Tests
```

Comment:
Please add a test that logs a `VostokLogEventWrapper` and verifies that underlying Vostok log receives exactly its underlying LogEvent.